app/javascript/channels/api/v1/race_channel.js

```diff
@@ -0,0 +1,270 @@
+const moment = require('moment')
+
+import consumer from '../../consumer'
+import { ts } from '../../../time'
+
+let raceSubscription
+let isConnceted = false
+
+document.addEventListener('turbolinks:load', () => {
+  if (!window.gon || !window.gon.race) {
+    return
+  }
+
+  raceSubscription = consumer.subscriptions.create({
+    channel:    'Api::V1::RaceChannel',
+    race_id:    window.gon.race.id,
+    race_type:  window.gon.race.type,
+    join_token: window.gon.race.join_token
+  }, {
+    connected() {
+      isConnceted = true
+      const alert = document.getElementById('disconnected-alert')
+      if (alert) {
+        alert.parentNode.removeChild(alert)
+      }
+
+      const forfeitTime = localStorage.getItem(`race-${window.gon.race.id}-forfeit-time`)
+      if (forfeitTime) {
+        this.forfeit(parseFloat(forfeitTime))
+        localStorage.removeItem(`race-${window.gon.race.id}-forfeit-time`)
+      }
+
+      const doneTime = localStorage.getItem(`race-${window.gon.race.id}-done-time`)
+      if (doneTime) {
+        this.done(parseFloat(doneTime))
+        localStorage.removeItem(`race-${window.gon.race.id}-done-time`)
+      }
+    },
+
+    disconnected() {
+      isConnceted = false
```

Comment:
My ability to typo words in unparalleled.

Reply:
Wow I just realized it's not just this occurrence, you typod every single occurrence in the exact same way including the declaration so this is Correct Code™ 😆

Reply:
I must've just autocompleted the wrong wrong spelling, I feel like there's no way that I could misspell something that many times in one try (although I type `destory` instead of `destroy` more than I would like to admit).